runtime/etc/Makefile.mli-common

```diff
@@ -0,0 +1,96 @@
+# Copyright 2004-2019 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Makefile.mli-common
+#
+
+include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
+
+# Set the CHPL_ZMQ_HOME variable.
```

Comment:
Misphrased -- this doesn't set the CHPL_ZMQ_HOME variable.  Maybe "Set ZMQ-related paths and options."